### PR TITLE
Add amp-analytics config to record CSI metrics for Doubleclick/AdSense impls.

### PR DESCRIPTION
**Pull request by warrengm**
This updates the Doubleclick and AdSense A4A impls to collect the triggers that were implemented in https://github.com/ampproject/amphtml/pull/11946.

### CODE REVIEW

File: ads/google/a4a/utils.js [hunk 1]
```diff
@@ -411,6 +424,96 @@ export function additionalDimensions(win, viewportSize) {
     innerHeight].join();
 };
 
+/**
+ * Returns a new CSI trigger.
+ * @param {!Object<string, string>} params
+ * @return {!JsonObject}
+ */
+function csiTrigger(on, params) {
```
Comment (glevitzky) [inline ads/google/a4a/utils.js]: Missing docstring for `on` param.

Comment (warrengm) [reply ads/google/a4a/utils.js]: Done. Is there presubmit to catch this?

**Comment by rsimha-amp** [comment]
This PR appears to have caused a test failure on master, that's consistently happening for other push builds too: https://travis-ci.org/ampproject/amphtml/builds/305424776

@keithwrightbos can you take a look and see if this is related?

**Comment by warrengm** [comment]
Hi Raghu, sorry I didn't notice it sooner. I reran the travis build and it passed. Is this still an issue?

**Comment by rsimha-amp** [comment]
@warrengm Seems like it's a flaky test. I'll keep an eye on it. Thanks for checking, and sorry about the false alarm!

**Pull request merged**